ccp/src/main/res/layout/layout_picker_dialog.xml

```diff
@@ -10,7 +10,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_margin="10dp"
-        android:text="Select country"
+        android:text="@string/select_country"
```

Comment:
Did you add "select_country", "search..." and "no_result_found" to strings XML file? because your [XML file](https://github.com/kenneth2008/CountryCodePickerProject/blob/master/ccp/src/main/res/values/strings.xml) is not showing these string entries.